tools/Dashboard/DevHome.Dashboard/Controls/WidgetControl.xaml.cs

```diff
@@ -169,18 +171,20 @@ private async void OnMenuItemSizeClick(object sender, RoutedEventArgs e)
 
                 // Set mark on new size.
                 _currentSelectedSize = menuSizeItem;
-                MarkSize(_currentSelectedSize);
+                var resourceLoader = new ResourceLoader("DevHome.Dashboard.pri", "DevHome.Dashboard/Resources");

```

Comment:
Can we just new up the ResourceLoader in the MarkSize() method instead of passing it in?